modules/layers/src/layers/editable-h3-layer.ts

```diff
@@ -0,0 +1,152 @@
+/* eslint-env browser */
+
+import { H3ClusterLayer } from '@deck.gl/geo-layers';
+import { ViewMode } from '@nebula.gl/edit-modes';
+import { polyfill } from 'h3-js';
+import EditableGeoJsonLayer from './editable-geojson-layer';
+import EditableLayer from './editable-layer';
+
+const DEFAULT_EDIT_MODE = ViewMode;
+const DEFAULT_H3_RESOLUTION = 9;
+
+const defaultProps = {
+  mode: DEFAULT_EDIT_MODE,
+
+  // EditableGeoJsonLayer
+  geoJsonData: {},
+  ...EditableGeoJsonLayer.defaultProps,
+
+  // h3 layer
+  h3Data: {},
+  filled: false,
+  stroked: true,
+  lineWidthScale: 1,
+  lineWidthMinPixels: 1,
+  lineWidthMaxPixels: Number.MAX_SAFE_INTEGER,
+  lineWidthUnits: 'pixels',
+  getHexagons: (d) => d.hexIds,
+  resolution: DEFAULT_H3_RESOLUTION,
+};
+
+export default class EditableH3Layer extends EditableLayer {
```

Comment:
I think it'd be great to have an example app demonstrating this layer as part of this PR.

Reply:
Good call. I'll put one together.

Reply:
You can use this as a starting point:
https://codesandbox.io/s/nebulagl-h3-hexagon-editing-xgntv?file=/index.js